### PR TITLE
cylc poll-all: no options given then poll all

### LINKED ISSUE #1177: cylc poll all submitted and running jobs
Currently `cylc poll` only works on jobs of tasks matching some name patterns on a given cycle. An ability to poll all submitted and running jobs in the suite would be useful, e.g. when a batch system for running most jobs of the suite has been restarted.

### PR DESCRIPTION
**Pull request by kaday**
Closes #1177

Tests added.

Polls all tasks when no options are given at the command line.

Note that in my environment including on master tests/cylc-show/03-clock-triggered-non-utc-mode.t fails.

### CODE REVIEW

File: lib/cylc/task_pool.py [hunk 1]
```diff
@@ -614,10 +614,9 @@ def no_active_tasks(self):
                 return False
         return True
 
-    def poll_tasks(self, ids):
+    def poll_tasks(self, ids=None):
         for itask in self.get_tasks(incl_runahead=False):
-            if itask.identity in ids:
```
Comment (hjoliver) [inline lib/cylc/task_pool.py]: Without this line, we can no longer poll individual tasks.

**Comment by hjoliver** [comment]
@kaday and @matthewrmshin - attempting to poll a single task or family will now result in all tasks being polled (easy to fix though). Otherwise looks good.

**Comment by kaday** [comment]
@hjoliver and @matthewrmshin  - fixed and works in my enviroment

**Comment by hjoliver** [comment]
One last problem - the test suite needs to run in UTC mode, otherwise the reference log comparison fails in other time zones.

**Comment by kaday** [comment]
@hjoliver Done and checked

**Pull request merged**